G/GnuTLS/build_tarballs.jl

```diff
@@ -29,25 +24,33 @@ if [[ ${target} == *darwin* ]]; then
     if [[ "${target}" == aarch64* ]]; then
         export CFLAGS="-mmacosx-version-min=11.0"
     else
-        export CFLAGS="-mmacosx-version-min=10.11"
+        export CFLAGS="-mmacosx-version-min=10.15"
```

Comment:
This is going to cut out many versions of macOS, is it really needed?

Reply:
10.11 gives a Mac build error, trying 10.12